pkg/plugins/platform/utils.go

```diff
@@ -128,32 +128,6 @@ func ComposeFileExists(app App) bool {
 	return true
 }
 
-// Cleanup will clean up ddev apps even if the composer file has been deleted.
-func Cleanup(app App) error {
```

Comment:
@beeradb I'd be fine with continuing to use this if it's bulletproof! Feel free to run with it if you feel like it's better to make it work. Thanks. (Not sending it your way because of lack of bandwidth but because of lack of background.)